project_status_report/report/project_status_report.xml

```diff
@@ -0,0 +1,110 @@
+                                </td>
+                                <td width="15%" style="text-align: center;vertical-align:middle;">Remarks</td>
+                                <td width="70%" style="border-style:solid;border-width:1px;"><t t-raw="evaluation_indicator[2]"/></td>
+                            </tr>
+                        </table>
+                        <br/>
+                    </t>
+
+                    <br/>
+
+                    <table width="100%">
+                        <tr>
+                            <td width="10%">Risks</td>
+                            <td width="90%" style="border-style:solid;border-width:1px;vertical-align:middle;"><t t-raw="o.risks"/></td>
+                        </tr>
+                    </table>
+
+                    <br/>
+
+                    <h3 style="border-style:solid;border-width:1px;">Project indicators</h3>
+
+                    <br/>
+
+                    <!--
+                        Here we want to display project indicators on 2 columns.
+
+                        By example, if project indicators are a, b, c, d, e, we will display
+                        a  b
+                        c  d
+                        e
+
+                        The foreach value will transform project indicators list to do that.
+                        Example: [a, b, c, d, e] ==> [[a, b], [c, d], [e]]
+                    -->
+                    <t t-foreach="[o.indicator_ids[i:i+2] for i in xrange(0, len(o.indicator_ids), 2)]" t-as="project_indicator_lines">
+                        <table width="100%">
+                            <tr>
+                                <!-- First project indicator in first column -->
+                                <t t-call="project_status_report.project_status_report_project_indicator_line">
+                                    <t t-set="project_indicator_line" t-value="project_indicator_lines[0]"/>
+                                </t>
+                                <!-- Space with columns -->
+                                <td width="10%"/>
+                                <!-- Second project indicator in second column (if we have second project indicator in line) -->
+                                <t t-if="len(project_indicator_lines) == 2">
+                                    <t t-call="project_status_report.project_status_report_project_indicator_line">
+                                        <t t-set="project_indicator_line" t-value="project_indicator_lines[1]"/>
+                                    </t>
+                                </t>
+                                <!-- Empty second column (if we don't have second project indicator in line) -->
+                                <t t-if="len(project_indicator_lines) == 1">
+                                    <td width="35%"/>
+                                    <td width="10%"/>
+                                </t>
+                            </tr>
```

Comment:
this foreach can be written that way

``` xml
<t t-foreach="o.indicator_ids" t-as="project_indicator_line">
    <t t-if="project_indicator_line_even">
        &lt;tr&gt;
    </t>
    <t t-call="project_status_report.project_status_report_project_indicator_line">
        <t t-set="project_indicator_line" t-value="project_indicator_line"/>
    </t>
    <t t-if="project_indicator_line_even">
        <t t-if="project_indicator_line_last">
            <td width=35%/>
        </t>
        <td width="10%"/>
    </t>
    <t t-if="project_indicator_odd">
        &lt;/tr&gt;
    </t>
</t>
```

foo_even : when element foo has an even index (0, 2, 4, ...)
foo_odd: when element foo has an odd index (1, 3, 5, ...)
foo_last: when element foo is the last of the list

did you try to use bootstrap.css grid here ?

Reply:
&lt;tr&gt; <- this should be supported to generate a tag which isn't closed